lte/gateway/c/core/oai/tasks/nas5g/src/M5GAuthenticationFailure.cpp

```diff
@@ -88,41 +82,36 @@
 }
 
 // Encoding Authentication Failure Message and its IEs
-int AuthenticationFailureMsg::EncodeAuthenticationFailureMsg(
-    AuthenticationFailureMsg* auth_failure, uint8_t* buffer, uint32_t len) {
+int AuthenticationFailureMsg::EncodeAuthenticationFailureMsg(uint8_t* buffer,
+                                                             uint32_t len) {
```

Comment:
No change in indentation has been made, only the removal of unnecessary field to avoid misuse of method and ease the comprehension of the code.
The indentation is automatically placed in the devcontainer environment.